src/System.Private.ServiceModel/tests/Scenarios/Bridge/Bridge/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Bridge")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Bridge")]
+[assembly: AssemblyCopyright("Copyright ©  2015")]
```

Comment:
I think we've avoided including AssemblyInfo.cs files since the copyrights are... weird. At least could you remove thing that could resemble a change in licencing terms for the Bridge.